Floydan/discord-pickup-bot
# Feature/add promote reaction static queue message

Added support for 📢 reaction in Pickup static message
clicking this reaction triggers a promote for the pickup

### To test
1. Create a pickup `!create [pickup name] <team size> <operator flags>`
2. View the pickup related static message in the `active-pickups` channel
3. Click the 📢 reaction and make sure you got a promote DM from the bot.

If you didn't get a promote make sure you have the role that `!subscribe` adds before promoting
